pkg/endpoint/providers/kube/announcements_test.go

```diff
@@ -0,0 +1,32 @@
+package kube
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
```

Comment:
Would be nice to have new tests written using golang `testing` lib if not too much of a overhead.

Reply:
I am going to write a test to cover all event types - PR coming